frontend/public/components/overview/index.tsx

```diff
@@ -486,11 +521,45 @@ class OverviewDetails extends SafetyFirst<OverviewDetailsProps, OverviewDetailsS
     };
   }
 
+  getBuildsForResource(buildConfig) {
+    const {builds} = this.props;
+    return getOwnedResources(buildConfig, builds.data);
+  }
+
+  getBuildConfigsForResource(resource) {
+    const {buildConfigs} = this.props;
+    const nativeTriggers = _.get(resource, 'spec.triggers');
+    const annotatedTriggers = JSON.parse(_.get(resource, ['metadata','annotations', TRIGGERS_ANNOTATION], '[]'));
+    const triggers = _.unionWith(nativeTriggers, annotatedTriggers, _.isEqual);
+    return _.flatMap(triggers, (trigger) => {
+      const from = trigger.from || _.get(trigger, 'imageChangeParams.from', {});
+      if ( from.kind !== 'ImageStreamTag') {
+        return [];
+      }
+      return _.reduce(buildConfigs.data, (acc, buildConfig) => {
+        if (_.get(buildConfig, 'spec.output.to.name') === from.name) {
```

Comment:
I think we need to check the namespace, too. IIRC, it is possible for a build in one namespace to push to an image stream in another namespace.

Reply:
So just check that the buildconfig and resource are in the same namespace?